main.go

```diff
@@ -18,11 +17,8 @@ func main() {
 			fmt.Println(datasource.WaterTemperatureToString(coldest))
 			fmt.Println(string(coldestJson))
 
-			fmt.Println("Check the response from the web server with http://localhost:8080/coldest")
-			http.HandleFunc("/coldest", func(w http.ResponseWriter, r *http.Request) {
-				datasource.LinkHandler(w, r, coldestJson)
-			})
-			http.ListenAndServe(":8080", nil)
+			datasource.StartServer("/coldest", coldestJson)
```

Comment:
Why is `coldestJson` sent into `StartServer`? I think this means that the server would constantly return the same response even if temperature changes and the response from the external api change